unicodetools/data/emoji/dev/ReadMe.txt

```diff
@@ -1,21 +1,21 @@
 # Unicode Emoji
-# © 2022 Unicode®, Inc.
+# © 2023 Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 
-This directory contains final data files for Unicode Emoji, Version 15.0
+This directory contains draft data files for Unicode Emoji, Version 15.1
 
-Public/emoji/15.0/
+Public/draft/emoji/
 
   emoji-sequences.txt
   emoji-zwj-sequences.txt
   emoji-test.txt
 
-The following related files are found in the UCD for Version 15.0
+The following related files are found in the UCD for Version 15.1
 
-Public/15.0.0/ucd/emoji/
+Public/draft/UCD/ucd/emoji/
```

Comment:
observation: it's curious how the segment `15.0.0` is being replaced by `draft/UCD`, given that `draft` seems like the version string that corresponds to `15.0.0`. In other words, it's a little asymmetrical, so slightly confusing, but not ultimately harmful.

Reply:
Yes, in the new scheme for alpha & beta (only), we have one single "draft" folder that contains the other pieces, and each of those without a version level. Documented in data-workflow.md.